src/objects.js

```diff
@@ -1,41 +1,65 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  }
+
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
+
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) {
+    return true;
+  } else {
+    return false
+  }
 };
 
 const getAges = people => {
-  // your code here
+  let peopleAges = [];
+  people.forEach(item => peopleAges.push(item.age));
+  return peopleAges;
```

Comment:
`map` would be a better method for this function, i.e. `return people.map(person => person.age);`